### PR TITLE
Various mappings in com/mojang/realmsclient/gui

### PR DESCRIPTION
**Pull request by PiTheGuy**

### CODE REVIEW

File: data/com/mojang/realmsclient/gui/RowButton.mapping [hunk 1]
```diff
@@ -8,9 +8,12 @@ CLASS com/mojang/realmsclient/gui/RowButton
 		ARG 1 poseStack
 		ARG 2 x
 		ARG 3 y
+		ARG 4 showTooltip
 	METHOD drawButtonsInRow (Lcom/mojang/blaze3d/vertex/PoseStack;Ljava/util/List;Lnet/minecraft/realms/RealmsObjectSelectionList;IIII)V
 		ARG 0 poseStack
 		ARG 1 buttons
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/gui/RowButton.mapping:14]: ARG 2 pendingInvitations

File: data/com/mojang/realmsclient/gui/screens/RealmsBrokenWorldScreen.mapping [hunk 2]
```diff
@@ -8,6 +8,9 @@ CLASS com/mojang/realmsclient/gui/screens/RealmsBrokenWorldScreen
 		ARG 1 slotIndex
 	METHOD drawSlotFrame (Lcom/mojang/blaze3d/vertex/PoseStack;IIIIZLjava/lang/String;IJLjava/lang/String;Z)V
 		ARG 1 poseStack
+		ARG 2 x
+		ARG 3 y
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/gui/screens/RealmsBrokenWorldScreen.mapping:12]: ARG 1 poseStack
ARG 2 x
ARG 3 y
ARG 4 mouseX
ARG 5 mouseY
ARG 6 isActiveNonMinigame
ARG 7 Text
ARG 8 slotName
ARG 9 templateID
ARG 10 templateImage

File: data/com/mojang/realmsclient/gui/screens/RealmsSelectFileToUploadScreen.mapping [hunk 3]
```diff
@@ -13,6 +13,8 @@ CLASS com/mojang/realmsclient/gui/screens/RealmsSelectFileToUploadScreen
 			ARG 2 levelSummary
 		METHOD renderItem (Lcom/mojang/blaze3d/vertex/PoseStack;III)V
 			ARG 1 poseStack
+			ARG 3 x
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/gui/screens/RealmsSelectFileToUploadScreen.mapping:17]: ARG 1 poseStack
ARG 2 index
ARG 3 x
ARG 4 y

File: data/com/mojang/realmsclient/gui/screens/RealmsSelectWorldTemplateScreen.mapping [hunk 4]
```diff
@@ -25,12 +25,18 @@ CLASS com/mojang/realmsclient/gui/screens/RealmsSelectWorldTemplateScreen
 			ARG 2 template
 		METHOD drawIcons (Lcom/mojang/blaze3d/vertex/PoseStack;IIIILjava/lang/String;Ljava/lang/String;Ljava/lang/String;)V
 			ARG 1 poseStack
+			ARG 2 x
+			ARG 3 y
 		METHOD drawImage (Lcom/mojang/blaze3d/vertex/PoseStack;IIIILcom/mojang/realmsclient/dto/WorldTemplate;)V
 			ARG 1 poseStack
+			ARG 2 x
```
Comment (Lanse505) [inline data/com/mojang/realmsclient/gui/screens/RealmsSelectWorldTemplateScreen.mapping:34]: ARG 1 poseStack
ARG 2 x
ARG 3 y
ARG 4 mouseX
ARG 5 mouseY
ARG 6 template

**Review by Lanse505 — CHANGES_REQUESTED** [review]
Looks good to me but there are 4 requested fixes

**Comment by PiTheGuy** [comment]
All changes have been applied

**Review by Lanse505 — APPROVED** [review]

**Pull request merged**